app/announcement/controllers/AnnouncementViewSet.py

```diff
@@ -156,17 +156,17 @@ def get_announcement_work_types(self, request):
         Returns:
             Response: Response containing list of announcement work types.
         """
-        return AnnouncementService.get_announcement_work_types()
+        return AnnouncementService.get_announcement_work_types(property_type_id)
 
     @action(
```

Comment:
Please set the detail parameter to True.

Reply:
`if you pass detail=True it means that that router will return you a single object whilst if you don't pass detail=True or pass detail=False it will return a list of objects.`
here we are returning a list